Core/clim-basic/clim-basic.asd

```diff
@@ -38,7 +38,7 @@
                  (:file "grafts"  :depends-on ("sheets" "ports"))
                  (:file "repaint" :depends-on ("sheets" "ports" "grafts" "events" "output"))))
    (:file "design" :depends-on ("decls" "protocol-classes" "utils" "geometry"))
-   (:file "X11-colors" :depends-on ("decls" "protocol-classes" "multiprocessing" "design" "geometry"))
+   (:file "colors" :depends-on ("design"))
```

Comment:
This change in filename should be reflected in `Tools/generated-named-colors.lisp`. In fact, that tool should be changed to generate a comment mentioning that the file has been generated and by what it has been generated.

Reply:
I've added comment in generated file. I've also squashed rename with creation of the module. it is a minor name change so there is no point in having a separate commit.